docs/01_nodeos/06_logging/20_third_party_logging/20_zipkin_tracer.md

```diff
@@ -9,7 +9,7 @@ The `Zipkin service` is a [distributed tracing system](https://zipkin.io/). It h
 
 ### How To Enable Zipkin Tracer
 
-EOSIO makes available `Zipkin tracer` through the [core `chain_plugin`](https://developers.eos.io/manuals/eos/v2.0/nodeos/plugins/chain_plugin/index/?query=chain%20plugin&page=1#gatsby-focus-wrapper). To enable the `Zipkin tracer` you must set the `telemetry-url` parameter for the `chain_plugin`. There are two additional parameters you can set: `telemetry-service-name` and `telemetry-timeout-us`. All three available parameters are detailed below:
+EOSIO makes available `Zipkin tracer` through the [core `chain_plugin`](https://developers.eos.io/manuals/eos/v2.1/nodeos/plugins/chain_plugin/index/?query=chain%20plugin&page=1#gatsby-focus-wrapper). To enable the `Zipkin tracer` you must set the `telemetry-url` parameter for the `chain_plugin`. There are two additional parameters you can set: `telemetry-service-name` and `telemetry-timeout-us`. All three available parameters are detailed below:
```

Comment:
@iamveritas this link to chain plugin should be relative (same repo, docs scope): `../../03_plugins/chain_plugin/index.md`, then whether it's v2.0, v2.1, or release is irrelevant. Filed issue #9804 to address this and two other link problems.